.kokoro/presubmit/integration.cfg

```diff
@@ -14,12 +14,12 @@ env_vars: {
 # TODO: remove this after we've migrated all tests and scripts
 env_vars: {
   key: "GCLOUD_PROJECT"
-  value: "gcloud-devel"
+  value: "java-docs-samples-testing"
```

Comment:
It did for most clients. Container and BQConnection failed. 
I was testing to check if JDST will perform better.

Turns out gcloud-devel is better since we use it for all our IT tests.
reverting it now.